core/trino-main/src/test/java/io/trino/spiller/TestFileSingleStreamSpiller.java

```diff
@@ -65,21 +63,12 @@ public class TestFileSingleStreamSpiller
     private static final List<Type> TYPES = ImmutableList.of(BIGINT, DOUBLE, VARBINARY);
 
     private final ListeningExecutorService executor = listeningDecorator(newCachedThreadPool());
-    private File spillPath;
```

Comment:
You could also remove `TestInstance(PER_CLASS)` (instead)

Reply:
Yeah - but there is no benefit of keeping this path as class field IMO